packages/react-reconciler/src/ReactFiberUnwindWork.js

```diff
@@ -205,9 +205,9 @@ function throwException(
     workInProgress = returnFiber;
     do {
       if (workInProgress.tag === SuspenseComponent) {
-        const state = workInProgress.memoizedState;
-        const didTimeout = state !== null && state.didTimeout;
-        if (!didTimeout) {
+        const fallback = workInProgress.memoizedProps.fallback;
+        const didTimeout = workInProgress.memoizedState;
+        if (!didTimeout && workInProgress.memoizedProps.fallback !== undefined) {
```

Comment:
Just noticed this. Looks like an oversight that you read from `workInProgress.memoizedProps.fallback` twice rather than use the `fallback` const.

Reply:
Seems like this code already changed.